core/blockchain.go

```diff
@@ -219,15 +219,16 @@ type BlockChain struct {
 	//  * nil: disable tx reindexer/deleter, but still index new blocks
 	txLookupLimit uint64
 
-	hc            *HeaderChain
-	rmLogsFeed    event.Feed
-	chainFeed     event.Feed
-	chainSideFeed event.Feed
-	chainHeadFeed event.Feed
-	logsFeed      event.Feed
-	blockProcFeed event.Feed
-	scope         event.SubscriptionScope
-	genesisBlock  *types.Block
+	hc              *HeaderChain
```

Comment:
Seems your editor is converting space to tab or the othereway arround

Reply:
I just added new stateChangeFeed feed, but it was longer in symbols, than others and had to re position everything. I am not sure, it is space/tab problem 🤔